src/share/poudriere/common.sh

```diff
@@ -1414,6 +1414,13 @@ get_data_dir() {
 	fi
 
 	if [ -z "${NO_ZFS}" ]; then
+		# Prepare older installs to take advantage of inheritance
```

Comment:
We still have the problem of /jails not getting fixed for `atime=off` and `compression=lz4` removed in https://github.com/freebsd/poudriere/pull/814/files

Reply:
See update in https://github.com/freebsd/poudriere/commit/1c9357d08e35f2bb248533ce043b2e61920ebf9e